packages/load/tests/loaders/schema/schema-from-json.spec.ts

```diff
@@ -27,7 +27,7 @@ describe('Schema From Export', () => {
       expect(isSchema(schema)).toBeTruthy();
       const introspectionSchema = require('./test-files/githunt.json').__schema;
       for (const typeName in schema.getTypeMap()) {
-        if (!typeName.startsWith('_')) {
+        if (!typeName.startsWith('__')) {
```

Comment:
Note: I haven't run these tests, but I suspect this is the same category of bug in a different place. Feel free to undo this change if I'm wrong 🙏